### ISSUE TITLE
Would there be any interest in packaging some of these scripts up with PyInstaller and so on for demo purposes?

### ISSUE THREAD

**Issue by nelsonjchen** [chriskiehl/GooeyExamples#15]
https://github.com/nelsonjchen/gooey-pyinstaller-demo

I made this demo project last year that demos using Gooey with PyInstaller and building it on Azure Pipelines.

https://github.com/MahApps/MahApps.Metro/releases/tag/2.0.0-alpha.83

Do you think it might be helpful to release out exes/.app versions of these applications to give users quick samples of Gooey applications? The MahApps.Metro example I linked above could be considered a demo for their widget library.

**Comment by chriskiehl** [comment]
I welcome the idea as a separate project! You could definitely link to them in the main Gooey wiki if you want. 

I don't want to have them in the main repo here, as supporting executables across all environments is a large, _large_ maintenance undertaking.

**Comment by nelsonjchen** [comment]
It's been a long time since I made this issue. Thanks for responding. Gooey and PyInstaller is my favorite Python cocktail by far.

If GitHub CI/CD is configured, I am certain the "large, *large*" maintenance undertaking thing won't be a thing. Most of the grunt work is done in PyInstaller and I should be clear that prebuilt executables would probably be something only targeting relatively stable desktop platforms such as macOS and Windows. It's not a very large matrix. 

It's probably even less now than when I made this issue if we're using the now fixed multi-entrypoint functionality of PyInstaller. Should be literally two ZIP release artifacts. 

Anyway, when I have some time, I'll fork and try to do a show and not tell with a PR on that claim on the scope of the toil. Believe me, it won't be "large, *large*"! 😄
